extensions/panache/mongodb-panache/runtime/src/main/java/io/quarkus/mongodb/panache/PanacheQuery.java

```diff
@@ -0,0 +1,161 @@
+package io.quarkus.mongodb.panache;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+import io.quarkus.panache.common.Page;
+
+/**
+ * <p>
+ * Interface representing an entity query, which abstracts the use of paging, getting the number of results, and
+ * operating on {@link List} or {@link Stream}.
+ * </p>
+ * <p>
+ * Instances of this interface cannot mutate the query itself or its parameters: only paging information can be
+ * modified, and instances of this interface can be reused to obtain multiple pages of results.
+ * </p>
+ *
+ * @author Loïc Mathieu - loikeseke@gmail.com
```

Comment:
No `@author` tags please (please look for them and remove them all).

Reply:
done